packages/psss/src/api/queries/useReportsSubmitted.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { getCurrentPeriod } from '@tupaia/utils';
+import { getWeekNumberByPeriod } from '../../utils';
+import { useData } from './useData';
+
+export const useReportsSubmitted = countryCodes => {
```

Comment:
Sorry to be a stickler for naming, but I think it may be a bit more clear if we name this like `useConfirmedCountries` or something?

Since it seems we are actually interested in just the no. countries with confirmed data in this endpoint. There will be an endpoint eventually which returns the no. sites which have submitted reports in a country, which I think the `useReportsSubmitted` name fits to better?

Reply:
That's a good point. What do you think about combining my `useReportsSubmitted` into `useConfirmedWeeklyReport`? That way it would be drier but a bit less easy to follow.

Reply:
I like that you've broken it out into a separate function actually. Even if they use the same query, I like the returned object here.